### PR TITLE
Add Dive into Machine Learning

### PR DESCRIPTION
**Pull request by floer32**
See if maybe someone wants to check out my help-wanted issues :) These are generally issues where I would appreciate a contribution from someone with some experience/expertise.

### CODE REVIEW

File: _data/projects/Dive_into_Machine_Learning.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+desc: Dive into Machine Learning with Python Jupyter notebook and scikit-learn
+site: https://github.com/hangtwenty/dive-into-machine-learning
+tags:
+- machine-learning
+- ML
+- python
+- scikit-learn
+- data-science
+upforgrabs:
+  name: help%20wanted
```
Comment (shiftkey) [inline _data/projects/Dive_into_Machine_Learning.yml]: To be honest, I'm not 100% sure whether we'll encode this value when displaying it, but you can totally set `help-wanted` here instead

Comment (floer32) [inline _data/projects/Dive_into_Machine_Learning.yml]: @shiftkey 

Oh, haha, good catch ... I wasn't sure if it need to exactly match, this makes sense though, updated to `help-wanted`

**Comment by floer32** [comment]
Hey, sorry to bother, did I fix to satisfaction?

**Comment by shiftkey** [comment]
@hangtwenty :thumbsup:

**Comment by floer32** [comment]
Cheers :)

**Pull request merged**